.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+    commit-message:
+      prefix: "CI"
```

Comment:
After the first PR was opened, I think we should be consistent for all the automated stuff. Either Build: Dependencies: CI: as prefix for all automated PRs.

Reply:
My thinking was that dependabot touches CI-only stuff (i.e. workflow dependencies) while the other job touches things which are related to our actual build logic. Therefore, I had chosen different prefixes.

If we want to have a single prefix for all, I'd go with *Build:*. aqt, Qt, etc. do run in CI, but not solely there, so that would sound wrong.

Reply:
I think the real problem is that we haven't agreed on a standard (i.e. when to use CI:, Build:, Autobuild, ...) Probably agreeing gives benefit but also more maintenance/thinking on our side.

Reply:
Agreed.  I don't really care if something is "CI" or "Build" or "Autobuild".  If the build breaks, the build breaks.  I'm _certain_ (almost) no client or server user cares, either! 😄 So the CHANGELOG entries really can be grouped, so that most people can just skip the lot when they hit them.

If I had to pick, I'd go for "Build" because it covers most ground.

Reply:
Change to *Build* added here: #2803.
I'll manually edit the titles (which are used when no CHANGELOG line is found) of the previous PRs.